README.md

```diff
@@ -53,6 +53,11 @@ Several reasons:
 * Everything is done in very few passes without expensive data transformations
 * Code is written with speed in mind, and tries to avoid unnecessary allocations
 
+## Documentation
+
+* [architecture](docs/architecture.md)
+* [中文文档](http://docs.breword.com/evanw-esbuild/)
```

Comment:
Why not adding into the current repo?

Reply:
Because translation docs my be outdated with the time going.

As follow suggestion from @evanw , add a link is better, and i will specific the version of the source code that translation docs represents.